db/seeds.rb

```diff
@@ -97,503 +65,390 @@ def terms_of_service_text
+SEMI = ENV["semi"].present?
+NO_PICS = ENV["no_pics"].present? || ENV["fast"].present?
 
-if ENV["minimal"]
-  entered_num = 3
-else
-  entered_num = ask_for_records("CheckoutProcedure")
+if MINIMAL
+  puts "Minimal mode activated. Please wait..."
 end
 
-unless entered_num == 0
-  if entered_num.integer? && entered_num > 0
-    progress = ProgressBar.create(format: progress_str, total: entered_num)
-    checkout_procedure = entered_num.times.map do
-      progress.increment
-      CheckoutProcedure.create! do |cho|
-        cho.step = Faker::HipsterIpsum.sentence
-        cho.equipment_model_id = equipment_model.flatten.sample.id
-      end
+t1 = Time.now
+
+# check to see if a superuser exists
+if User.where('role = ?', 'superuser').empty?
+  User.destroy_all
+  u = User.new
+  u.first_name = "Donny"
+  u.last_name = "Darko"
+  u.phone = "6666666666"
+  u.email = "email@email.com"
+  u.affiliation = "Your Mother"
+  u.role = 'superuser'
+  u.view_mode = 'superuser'
+  u.username = 'dummy'
+  u.save
+
+  if MINIMAL || SEMI
+    if ENV['CAS_AUTH']
+      printf '(NetID)'
+      prompt_field(u, :username)
+    else
+      u.username = u.email
+      u.password = 'passw0rd'
+      u.password_confirmation = u.password
+      u.save
     end
-    puts "\n#{entered_num} checkout procedures successfully created!"
   else
-    puts "\nPlease enter a whole number greater than 0."
-    entered_num = STDIN.gets.chomp.to_i
+    puts 'We need to create an account for you first. Please enter the following info:'
+    prompt_field(u, :first_name)
+    prompt_field(u, :last_name)
+    prompt_field(u, :phone)
+    prompt_field(u, :email)
+    prompt_field(u, :affiliation)
+    if ENV['CAS_AUTH']
+      printf '(NetID)'
+      prompt_field(u, :username)
+    else
+      u.username = u.email
+      puts "Temp Password:"
+      u.password = STDIN.gets
+      u.password_confirmation = u.password
+      u.save
+    end
   end
+
 end
 
-# Blackout Date generation
+
+# AppConfig generation
 # ============================================================================
 
-if ENV["minimal"]
-  entered_num = 0
-else
-  entered_num = ask_for_records("Blackout Dates")
+if AppConfig.count == 0
+
+  ac = AppConfig.new
+  ac.terms_of_service = TOS_TEXT
+  ac.reservation_confirmation_email_active = false
+  ac.overdue_checkin_email_active = false
+  ac.site_title = "Reservations"
+  ac.upcoming_checkin_email_active = false
+  ac.admin_email = "admin@admin.com"
+  ac.department_name = "Department"
+  ac.contact_link_location = "link"
+  ac.home_link_text = "home_link"
+  ac.home_link_location = "Canada"
+  ac.deleted_missed_reservation_email_body = DELETED_MISSED_RES_EMAIL
+  ac.default_per_cat_page = 10
+  ac.request_text = ''
+  ac.upcoming_checkin_email_body = UPCOMING_CHECKIN_RES_EMAIL
+  ac.overdue_checkin_email_body = OVERDUE_RES_EMAIL_BODY
+  ac.save
+
+  unless MINIMAL || SEMI
+    puts "We need to setup application settings:"
```

Comment:
Do we want to have some more specific / explanatory prompts for these settings? I guess they're mostly self-explanatory although `contact_link_location` is pretty confusing, I think (isn't it the e-mail to which contact messages are sent?).